DGC documentation (#1397)
corona-warn-app/cwa-server: docs/DISTRIBUTION.md

140 Temporary Exposure Keys (configurable by the `shifting-policy-threshold` property). Where there are less
keys available to distribute in a specific distribution run, these keys are shifted to a succeeding export,
until the threshold minimum is fulfilled.

## Digital Green Certificate

A Digital Green Certificate is a digital proof that a person has either been vaccinated against COVID-19, received a negative test result or recovered from COVID-19.

There are three different value sets to consume:
 - vaccine-prophylaxis.json - Vaccine or prophylaxis
 - vaccine-medicinal-product.json - Vaccine medicinal product

>  - vaccine-mah-manf.json - Marketing Authorization Holder

Reviewer: Before each of these bullets point you would need to remove the whitespace.:)

> They can be found in the dgc folder. ([here](https://github.com/corona-warn-app/cwa-server/tree/5e47a2e485585043a05ec4173204dd020c757585/services/distribution/src/main/resources/dgc))

Reviewer 2: ```suggestion
They can be found in the [dgc folder](https://github.com/corona-warn-app/cwa-server/tree/5e47a2e485585043a05ec4173204dd020c757585/services/distribution/src/main/resources/dgc).
```

The [`DigitalGreenCertificateToProtobufMapping`](/services/distribution/src/main/java/dgc/coronawarn/server/services/distribution/dgc/DigitalGreenCertificateToProtobufMapping.java)
is responsible for reading the values, using the [`DistributionServiceConfig`](/services/distribution/src/main/java/dgc/coronawarn/server/services/distribution/config/DistributionServiceConfig.java) and transforming the files into protobuffs.

> At the end of the process there is created this structure: ehn-dgc/{supportedLanguage}/value-sets.

Reviewer 2: ```suggestion
At the end of the process these URLs are created to allow retrieving the protobuf files: `ehn-dgc/{supportedLanguage}/value-sets`.
```

> The supported languages are configurable, for now: supportedLanguage: DE, EN, BG, PL, RO, TR. (more details [here](https://github.com/corona-warn-app/cwa-server/blob/5e47a2e485585043a05ec4173204dd020c757585/services/distribution/src/main/resources/application.yaml#L208)).

Reviewer 2: ```suggestion
The supported languages are [configurable](https://github.com/corona-warn-app/cwa-server/blob/5e47a2e485585043a05ec4173204dd020c757585/services/distribution/src/main/resources/application.yaml#L208), for now they are: DE, EN, BG, PL, RO, TR.
```
